clients/js/src/cmds/parse.ts

```diff
@@ -7,8 +7,22 @@ export const builder = (y: typeof yargs) => {
   return y.positional("vaa", {
     describe: "vaa",
     type: "string",
+  })
+  .option("json", {
+    alias: "j",
+    describe: "Print results in json format",
+    type: "boolean",
+    default: false,
   });
 };
+// eslint-disable-next-line @typescript-eslint/no-redeclare
```

Comment:
instead of redeclaring `BigInt`'s `toJSON` implementation, maybe we could use a replacer in `JSON.stringify`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify

Reply:
I figured the mozilla folks knew better than me and [used the first approach they suggested](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#use_within_json). I can make it use a replacer.